microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Simulate a Window Controls Overlay in a PWA's title bar area
```

Comment:
```suggestion
title: Simulate the Window Controls Overlay API without installing your PWA
```

Window Controls Overlay is really the name of the PWA feature (or API) here. So the phrase "a Window Controls Overlay" didn't make a lot of sense. The benefit of this new DevTools feature is to simulate WCO being used without having to first install the app locally. Feel free to propose different phrasing.

Reply:
globally updated title